cypress/e2e/Class01.cy.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="Cypress" />
+
+describe('Validate EngageSphere website', ()=> {
```

Comment:
Que tal assim?

```suggestion
describe('EngageSphere', ()=> {
```

Lembre da aula 1. `describe` e `context` devem ser concisos.
Deixe descrições mais longas para os casos de testes (blocos `it`).